test/Integration/SqlOutputBindingIntegrationTests.cs

```diff
@@ -46,6 +45,9 @@ public void AddProductTest(int id, string name, int cost, SupportedLanguages lan
         [SqlInlineData(1, "Test", 5)]
         [SqlInlineData(0, "", 0)]
         [SqlInlineData(-500, "ABCD", 580)]
+        // Currently PowerShell returns null if the output
```

Comment:
Currently PowerShell returns null when the parameter for name is an empty string